### DISCUSSION TITLE
Congratulations to Victoria Nduka🎉🎉🎉

### DISCUSSION THREAD

**Discussion by gbengaoluwadahunsi** [camicroscope/caMicroscope#963] [General]
I want to use this medium to congratulate @nwanduka for being selected as the intern for caMicroscope outreachy project 2024.

We did wonderfully well, it's what it is and one out of many was picked.

I wish us all the best in our endeavours.

Thank you @birm for the opportunity to contribute.

I personally will  still go ahead and do the clean up contributions that I alighted in my final application.

![Screenshot_20240503-091711.png](https://github.com/camicroscope/caMicroscope/assets/77133593/08b4f984-0377-40f9-87d0-a3aa52154e17)

![Screenshot_20240503-091738.png](https://github.com/camicroscope/caMicroscope/assets/77133593/9d990b2e-c9c2-4a4c-adfe-7c911d6b84ba)

**Comment by nwanduka** [comment]
Thank you so much @gbengaoluwadahunsi. I'm really excited about this opportunity. It was a pleasure learning and contributing alongside everyone. Your contributions to the project have been invaluable and will, no doubt, give me a head start during the internship. Thanks again.😊

**Reply by gbengaoluwadahunsi** [reply]
All the best.

**Reply by DIANAAJAMBO** [reply]
I am definitely late to the party but i still had to congratulate you @nwanduka . i was new to accessibility but  learnt a lot from your contribution. All the best

**Reply by nwanduka** [reply]
Aww.. Thanks so much, @DIANAAJAMBO.
